apps/meteor/client/components/Page/PageHeader.tsx

```diff
@@ -10,7 +10,7 @@ type PageHeaderProps = {
 	title: ReactNode;
 	onClickBack?: () => void;
 	borderBlockEndColor?: string;
-};
+} & ComponentProps<typeof Box>;
```

Comment:
```suggestion
}
```
if we move forward with the `backgroundColor` on this component I would suggest removing the borderBlockEndColor, because the `ComponentProps<typeof Box>` would be enough, otherwise there's no need for this